ui/hooks/gasFeeInput/useMaxFeePerGasInput.js

```diff
@@ -34,6 +34,7 @@ const getMaxFeePerGasFromTransaction = (transaction) => {
  *  method to update the setMaxFeePerGas.
  */
 export function useMaxFeePerGasInput({
+  EIP_1559_V2,
```

Comment:
Maybe this should be called `supportsEIP1559V2`? I think there are other files where we pass a similar prop. Otherwise it looks like this is a constant and never changes.

Reply:
Yep I was waiting to rebase it over develop, fixed now